test/web-platform-tests/to-upstream/dom/nodes/move-me-plz.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<!-- TODO: fix below -->
+<title>not sure what to name this or where to put it</title>
+<!-- TODO: fix below -->
+<link rel=help href="https://dom.spec.whatwg.org/#dom-element-hasattribute">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<!-- TODO: find out what this is -->
+<span data-e2="2" data-F2="3" id="t"></span>
```

Comment:
This is just test data that a particular test you're copying from used. You can omit it in your test.